decidim-core/app/packs/src/decidim/input_emoji.js

```diff
@@ -29,9 +32,20 @@ export default function addInputEmoji() {
       wrapper.appendChild(elem);
       wrapper.appendChild(btnContainer);
 
-      btnContainer.addEventListener("click", () => picker.togglePicker(btnContainer))
+      btnContainer.addEventListener("click", handlerPicker);
```

Comment:
Sorry missed this in the earlier review but I think you don't need to change this, right?

The less things you change in PRs, the better.

Reply:
I needed to name the function because i can't remove the event listener without the name of the function

Reply:
Oh right, got it. Never mind then!